Project4/styles.css

```
@@ -0,0 +1,151 @@
+@import url(https://fonts.googleapis.com/css?family=Montserrat);
+
+$nav-height: 60px;
+h1 h2 h3 {
+  color: #ffc400;
+}
+body {
+  text-align:center;
+  margin:0px;
```

Values of zero do not need units:

```suggestion
  margin:0;
```